pkgs/development/lua-modules/generated-packages.nix

```diff
@@ -326,6 +366,76 @@ lua-term = buildLuarocksPackage {
     };
   };
 };
+luaevent = buildLuarocksPackage {
+  pname = "luaevent";
+  version = "0.4.6-1";
+
+  src =  fetchurl {
+      url    = https://luarocks.org/luaevent-0.4.6-1.src.rock;
+      sha256 = "0chq09nawiz00lxd6pkdqcb8v426gdifjw6js3ql0lx5vqdkb6dz";
+   };
+  disabled = ( luaOlder "5.1");
+  propagatedBuildInputs = [lua  ];
+  buildType="builtin";
+
+  meta = {
+    homepage = "https://github.com/harningt/luaevent";
+    description="libevent binding for Lua";
+    license = {
+      fullName = "MIT";
+    };
+  };
+};
+luabitop = buildLuarocksPackage {
+  pname = "luabitop";
+  version = "1.0.2-3";
+
+  knownRockspec = ( fetchurl {
+      url    = https://luarocks.org/luabitop-1.0.2-3.rockspec;
+      sha256 = "07y2h11hbxmby7kyhy3mda64w83p4a6p7y7rzrjqgc0r56yjxhcc";
+   }).outPath;
+
+   src =  fetchgit ( removeAttrs (builtins.fromJSON ''{
+  "url": "git://github.com/LuaDist/luabitop.git",
+  "rev": "81bb23b0e737805442033535de8e6d204d0e5381",
+  "date": "2013-02-18T16:36:42+01:00",
+  "sha256": "0lsc556hlkddjbmcdbg7wc2g55bfy743p8ywdzl8x7kk847r043q",
+  "fetchSubmodules": true
+}
```

Comment:
I wonder if there is an easy way to improve generated formatting here — this point is a large «wait what» when reading.

Reply:
I didn't want to write my own tool so I use the output of `nix-prefetch-url/git`. This is my cry out for help:
https://discourse.nixos.org/t/generate-nix-code-to-fetch-package-from-url/1941

Anyway as this is generated, it's easy to fix afterwards, luarocks-nix is a huge mess now I would need to remove all the comments first so that others can fix it. I would also like to have everything sorted alphabetically as well.

Reply:
I can remove it from the list if it's a problem

Reply:
Nope, not an immediate problem, just a thing to keep in mind.

I wonder if this file would get better or worse from just feeding through `indent`.